setup.py

```diff
@@ -47,7 +50,7 @@
     },
     # Packaging
     package_dir={"": "src"},
-    packages=find_packages("src"),
+    packages=find_packages("src") + namespace_packages,
```

Comment:
Do we need to add namespace_package and use find_namespace_packages?

https://setuptools.readthedocs.io/en/latest/setuptools.html#namespace-packages
https://setuptools.readthedocs.io/en/latest/setuptools.html#find-namespace-packages

https://setuptools.readthedocs.io/en/latest/pkg_resources.html?#namespace-package-support

Reply:
the find_namespace_packages is a good suggestion


 pkg_resources.declare_namespace approach is no longer recommended as per https://packaging.python.org/guides/packaging-namespace-packages/#pkg-resources-style-namespace-packages